index.bs

```diff
@@ -4590,3 +4763,14 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution Scope ## {#attribution-scope}
+
+*This section is non-normative.*
+
+Abusive actors can exploit the attribution scope feature by registering "ghost navigation sources" instead of event sources: During source registration, they can redirect to themselves and register [=attribution source/attribution scope limit=] - 1 navigations. This allows them to choose more complex configurations (more bits) for their ghost registrations.
+
+To prevent this, we can limit the number of source registrations per reporting origin per navigation for sources that use the attribution scope feature.
+
+Proposed mitigation: Limit to 1 navigation-source with non-empty [=attribution source/attribution scopes=] per registration per reporting origin.
+
```

Comment:
Let's also link to https://github.com/WICG/attribution-reporting-api/issues/1128, which applies a limit to this regardless of the scopes feature.